l3kernel/testfiles/m3text006.tlg

```diff
@@ -0,0 +1,540 @@
+This is a generated file for the LaTeX (2e + expl3) validation system.
+Don't change this file in any respect.
+Author: Joseph Wright
+============================================================
+TEST 1: Grapheme mapping
+============================================================
+TRUE
+TRUE
+TRUE
+TRUE
+TRUE
+TRUE
+TRUE
+TRUE
```

Comment:
if that ever says something else you have probably a hard time to figures out which TRUE is which :-)

Reply:
True, and but when it does that it's quite easy to add a `\TYPEOUT` and check

Reply:
Ah, I actually left that in the test, so a failure will print the text which is wrong.

Reply:
ok then :-)